{"name": "User ID does not exist", "status": "passed", "steps": [{"name": "Given the DELETE header \"app-id\" is \"665acdd5b842089f2e37b051\"", "status": "passed", "start": 1717322800868, "stop": 1717322800869}, {"name": "And the DELETE request user ID is \"60d0fe4f5311236168a109f1\"", "status": "passed", "start": 1717322800869, "stop": 1717322800869}, {"name": "When I make a DELETE request to it", "status": "passed", "start": 1717322800869, "stop": 1717322801196}, {"name": "Then the DELETE status code should be 404", "status": "passed", "start": 1717322801196, "stop": 1717322801196}, {"name": "And the DELETE response should contain \"error\" with value \"RESOURCE_NOT_FOUND\"", "status": "passed", "start": 1717322801196, "stop": 1717322801197}], "start": 1717322800866, "stop": 1717322801197, "uuid": "727d2bdb-faec-4755-983b-a85d12a1fbc4", "historyId": "14137fc192ebb1ad0dc53dbfdb3c532b", "fullName": "Api testing for delete endpoint: User ID does not exist", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "Api testing for delete endpoint"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}